src/jmbitcoin/secp256k1_main.py

```diff
@@ -22,8 +22,9 @@
 N = 115792089237316195423570985008687907852837564279074904382605163141518161494337
 
 BTC_P2PK_VBYTE = {"mainnet": b'\x00', "testnet": b'\x6f', "signet": b'\x6f',
-    "regtest": 100}
-BTC_P2SH_VBYTE = {"mainnet": b'\x05', "testnet": b'\xc4', "signet": b'\xc4'}
+    "regtest": 100, "testnet4": b'\x6f'}
+BTC_P2SH_VBYTE = {"mainnet": b'\x05', "testnet": b'\xc4', "signet": b'\xc4',
+    "testnet": b'\xc4'}
```

Comment:
Haven't yet tested, but there seems to be mistake (could we catch it with the tests?)

```suggestion
BTC_P2SH_VBYTE = {"mainnet": b'\x05', "testnet": b'\xc4', "signet": b'\xc4',
    "testnet4": b'\xc4'}
```

Reply:
Yes, it was a typo. I'm sorry.

Reply:
It can be tested on wallets with `p2sh-p2wpkh` SetWit